core/templates/domain/story/editable-story-backend-api.service.spec.ts

```diff
@@ -16,49 +16,35 @@
  * @fileoverview Unit tests for EditableStoryBackendApiService.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
-
-import { TranslatorProviderForTests } from 'tests/test.extras';
-
-require('domain/editor/undo_redo/undo-redo.service.ts');
-require('domain/story/editable-story-backend-api.service.ts');
-require('services/csrf-token.service.ts');
-
-describe('Editable story backend API service', function() {
-  var EditableStoryBackendApiService = null;
-  var sampleDataResults = null;
-  var $rootScope = null;
-  var $scope = null;
-  var $httpBackend = null;
-  var UndoRedoService = null;
-  var CsrfService = null;
-
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(
-    angular.mock.module('oppia', TranslatorProviderForTests));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
-    }
-  }));
+import { EditableStoryBackendApiService } from
+  'domain/story/editable-story-backend-api.service.ts';
```

Comment:
Drop the ".ts" part (see other files).
Ditto below.

Reply:
Done.